frontend/jest.config.js

```diff
@@ -50,7 +50,7 @@ const config = {
     '^@studio/components': path.join(__dirname, 'libs/studio-components/src/$1'),
     '^@studio/pure-functions': path.join(__dirname, 'libs/studio-pure-functions/src/$1'),
   },
-  testRegex: '(\\.(test|spec))\\.(ts|tsx)$',
+  testRegex: '(\\.(test))\\.(ts|tsx)$',
```

Comment:
I have checked that we do not have any unit-tests which follows the `<filename>.spec.ts` pattern. I needed to change this since we are using `.spec` for our playwright tests.